packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -246,11 +246,18 @@ export class VerifiedFetch {
     let prevModificationId = context.modified
 
     while (passCount < maxPasses) {
-      this.log(`Starting pipeline pass #${passCount + 1}`)
-      passCount++
+      if (context.options?.signal?.aborted === true) {
```

Comment:
Same comment as https://github.com/ipfs/helia-verified-fetch/pull/241/files#r2104680279 - again there's only one use of `await` in this method so if the signal wasn't aborted when it was passed into this method, execution has to have been suspended by the `await plugin.handle(context)` when the event fired.